lib/tasks/after_tasks.rake

```diff
@@ -740,6 +740,76 @@ namespace :After do
     WorkIndexer.create_mapping
   end
 
+  desc "Fix works imported with a noncanonical Teen & Up Audiences rating tag"
+  task(fix_teen_and_up_imported_rating: :environment) do
+    borked_rating_tag = Rating.find_by!(name: "Teen & Up Audiences")
+    canonical_rating_tag = Rating.find_by!(name: ArchiveConfig.RATING_TEEN_TAG_NAME)
+    works_using_tag = borked_rating_tag.works
+    invalid_works = []
+    works_using_tag.each do |work|
+      work.ratings << canonical_rating_tag
+      work.ratings = work.ratings - [borked_rating_tag]
+      invalid_works << work.id if work.save == false
+    end
+
+    unless invalid_works.empty?
+      puts "The following works failed validations and could not be saved:"
+      puts invalid_works.join(", ")
+      STDOUT.flush
+    end
+
+    puts "Converted #{borked_rating_tag.name} rating tag on #{works_using_tag.size - invalid_works.size} works"
+    STDOUT.flush
+  end
+
+  desc "Clean up noncanonical rating tags"
```

Comment:
I checked on production, and looks like we only have one non-canonical rating ("Teen & Up Audiences") which is fixed by `fix_teen_and_up_imported_rating`. I think we can leave out this task.

Reply:
Sorry, I checked again, and we have multiple non-canonical ratings on the staging environment. So we do want to keep this task after all.

Are you around to resurrect the task removed in fbd892d740e29891bfb0454f959b5d97e0de2665? If you're busy, I can do that and list you as a co-author.

Reply:
I won't be able to get to this for a couple of days, so feel free to go ahead if you want to get in sooner.

Reply:
👍, I've opened #4047.